# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xpcom/nsAccessibleRelation.cpp
# Commit: 86da8f2a28ea
# Full Hash: 86da8f2a28ea1992671d08dfb56c881a1a82daa5
# Author: James Teh <jteh@mozilla.com>
# Date: 2022-08-04 04:34:13
# Regressor Bug: 1781536
# File Overlap Count: 1
# Description:
#   Bug 1781536 part 3: Move RelationByType to base Accessible. r=morgan
#   
#   This required changing the RemoteAccessible implementations to return Relation instead of an array of RemoteAccessible.
#   Platform implementations have been updated to use the unified method where appropriate.
#   
# ==============================================================================

diff -r 95ae96802dbe -r 86da8f2a28ea accessible/xpcom/nsAccessibleRelation.cpp
--- a/accessible/xpcom/nsAccessibleRelation.cpp	Wed Aug 03 23:58:51 2022 +0000
+++ b/accessible/xpcom/nsAccessibleRelation.cpp	Wed Aug 03 23:58:51 2022 +0000
@@ -23,16 +23,6 @@
   }
 }
 
-nsAccessibleRelation::nsAccessibleRelation(
-    uint32_t aType, const nsTArray<RemoteAccessible*>* aTargets)
-    : mType(aType) {
-  mTargets = do_CreateInstance(NS_ARRAY_CONTRACTID);
-  for (uint32_t idx = 0; idx < aTargets->Length(); ++idx) {
-    mTargets->AppendElement(
-        static_cast<nsIAccessible*>(ToXPC(aTargets->ElementAt(idx))));
-  }
-}
-
 nsAccessibleRelation::~nsAccessibleRelation() {}
 
 // nsISupports